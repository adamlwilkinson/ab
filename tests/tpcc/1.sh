pwd

echo "$0 $-"
basename $0
dirname $0

