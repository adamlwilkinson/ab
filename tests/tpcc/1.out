*************************************
*** ###easy### TPC-C Data Loader  ***
*************************************
<Parameters>
     [server]: 127.0.0.1
     [port]: 3306
     [DBname]: tpcc_w10
       [user]: root
       [pass]: 
  [warehouse]: 10
  [part(1-4)]: 1
     [MIN WH]: 1
     [MAX WH]: 10

1049, 42000, Unknown database 'tpcc_w10'
*************************************
*** ###easy### TPC-C Data Loader  ***
*************************************
<Parameters>
     [server]: 127.0.0.1
     [port]: 3306
     [DBname]: tpcc_w10
       [user]: root
       [pass]: 
  [warehouse]: 10
  [part(1-4)]: 1
     [MIN WH]: 1
     [MAX WH]: 10
TPCC Data Load Started...
Loading Item 
.................................................. 5000
.................................................. 10000
.................................................. 15000
.................................................. 20000
.................................................. 25000
.................................................. 30000
.................................................. 35000
.................................................. 40000
.................................................. 45000
.................................................. 50000
.................................................. 55000
.................................................. 60000
.................................................. 65000
.................................................. 70000
.................................................. 75000
.................................................. 80000
.................................................. 85000
.................................................. 90000
.................................................. 95000
.................................................. 100000
Item Done. 

...DATA LOADING COMPLETED SUCCESSFULLY.
*************************************
*** ###easy### TPC-C Data Loader  ***
*************************************
<Parameters>
     [server]: 127.0.0.1
     [port]: 3306
     [DBname]: tpccw2500
       [user]: root
       [pass]: 
  [warehouse]: 2500
  [part(1-4)]: 1
     [MIN WH]: 1
     [MAX WH]: 2500
TPCC Data Load Started...
Loading Item 
.................................................. 5000
.................................................. 10000
.................................................. 15000
.................................................. 20000
.................................................. 25000
.................................................. 30000
.................................................. 35000
.................................................. 40000
.................................................. 45000
.................................................. 50000
.................................................. 55000
.................................................. 60000
.................................................. 65000
.................................................. 70000
.................................................. 75000
.................................................. 80000
.................................................. 85000
.................................................. 90000
.................................................. 95000
.................................................. 100000
Item Done. 

...DATA LOADING COMPLETED SUCCESSFULLY.
